tests/lib/bm_zms/src/main.c

```diff
@@ -116,6 +116,12 @@ ZTEST_F(bm_zms, test_bm_zms_register)
 {
 	int err;
 
+	err = bm_zms_register(NULL, &bm_zms_test_handler);
+	zassert_true(err == -EINVAL, "bm_zms_register unexpected failure");
+
+	err = bm_zms_register(&fixture->fs, NULL);
+	zassert_true(err == -EINVAL, "bm_zms_register unexpected failure");
+
 	err = bm_zms_register(&fixture->fs, &bm_zms_test_handler);
 	zassert_true(err == 0, "bm_zms_register call failure");
```

Comment:
This file can probably be cleaned a bit like this. Could be left as a separate PR. 
```suggestion
	zassert_equal(-EINVAL, err);

	err = bm_zms_register(&fixture->fs, NULL);
	zassert_equal(-EINVAL, err);

	err = bm_zms_register(&fixture->fs, &bm_zms_test_handler);
	zassert_ok(err, "bm_zms_register call failure");
```